WordPress/src/main/res/values/strings.xml

```diff
@@ -558,6 +558,8 @@
     <string name="older_last_week">Older than a week</string>
     <string name="older_month">Older than a month</string>
     <string name="error_notification_open">Could not open notification</string>
+    <string name="ignore">Ignore</string>
+    <string name="push_auth_expired">The login request has expired. Log in to WordPress.com to try again.</string>
```

Comment:
Sure, sounds good. How about:

`The request has expired. Sign in to WordPress.com to try again."

Interestingly enough, WordPress.com uses 'Log In' but has a checkbox that says 'Stay signed in'
![screen shot 2015-06-03 at 9 42 31 am](https://cloud.githubusercontent.com/assets/789137/7965858/09dcf658-09d5-11e5-8119-226eca68cf37.png)